app/lib/cms/s3.rb

```diff
@@ -32,11 +32,15 @@ def credentials
       CMS::AwsCredentials.instance.credentials
     end
 
+    def hostname
+      config[:hostname].presence if enabled?
```

Comment:
maybe just update line 41 instead `config[param] if enabled?`. Perhaps all params should have same behavior.

Reply:
Updated in 28bea41.